spire/templates/apps-300A.yml

```diff
@@ -152,6 +152,8 @@ Resources:
         DovetailCountedKinesisStreamName: !Ref DovetailCountedKinesisStreamName
         DovetailRouterHosts: !Sub /prx/${EnvironmentTypeAbbreviation}/dovetail-analytics/DOVETAIL_ROUTER_HOSTS
         DovetailRouterApiTokens: !Sub /prx/${EnvironmentTypeAbbreviation}/dovetail-analytics/DOVETAIL_ROUTER_API_TOKENS
+        FrequencyDynamodbTableName: !Sub /prx/${EnvironmentTypeAbbreviation}/dovetail-analytics/FREQUENCY_DDB_TABLE
```

Comment:
These Parameter Store parameters are using a naming convention that predates the [current standard](https://github.com/PRX/internal/wiki/AWS:-Parameter-Store#spire), and the few parameters that remain from back then are [slated to be cleaned up](https://github.com/PRX/internal/issues/887#issuecomment-1538922226).

It would be ideal if we didn't add any new parameters that don't follow the current standard, but if it makes more sense to deal with this later, let's make sure they at least get added to that issue.